test/verify/check-networking-firewall

```diff
@@ -138,15 +152,22 @@ class TestFirewall(MachineCase):
         m = self.machine
 
         self.login_and_go("/network/firewall")
+        b.wait_in_text("#zones-listing", "Public")
+        b.wait_in_text("#zones-listing", "default")
 
         def removeService(service):
             b.click("tr[data-row-id='{}'] .btn.pficon-delete".format(service))
+            if b.is_present("#remove-services-dialog-body #zone-input input:not(:checked)"):
```

Comment:
This kind of dynamic behaviour makes tests less predictable and understandable. Please rather turn this into an explicit argument if you want to use it for diifferent cases.